src/test/java/com/epam/tc/hw2/tests/CrudEntities.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.tc.hw2.tests;
+
+import static com.epam.tc.hw2.trello.asserts.TrelloAssertProvider.assertThat;
+import static org.testng.Assert.assertEquals;
+
+import com.epam.tc.hw2.data.TestDataProviders;
+import com.epam.tc.hw2.trello.dto.BoardDto;
+import com.epam.tc.hw2.trello.dto.ListDto;
+import io.restassured.response.Response;
+import org.testng.Reporter;
+import org.testng.annotations.Test;
+
+public class CrudEntities extends InitTest {
+
+    @Test(dataProvider = "boardName",
+          dataProviderClass = TestDataProviders.class,
+          description = "Create board -> get board -> compare")
+    void serviceShouldCreateNewBoard(String name) {
+        var boardDraft = BoardDto.builder().name(name).build();
+        var remoteBoard = apiUnderTest.createBoard(boardDraft);
+        createdBoards.add(remoteBoard);
+        var getBoard = apiUnderTest.getBoard(remoteBoard);
+        assertEquals(getBoard, remoteBoard);
+        Reporter.log( "Test: serviceShouldCreateNewBoard complete", true );
+    }
+
+    @Test(dataProvider = "boardName",
+          dataProviderClass = TestDataProviders.class,
+          description = "Create board -> delete board -> check status")
+    void serviceShouldDeleteBoard(String name) {
+        var boardDraft = BoardDto.builder().name(name).build();
```

Comment:
maybe you could separate some repeating code for some function?

Reply:
Done